src/content/docs/get-started/index.mdx

```diff
@@ -237,168 +219,67 @@ The Code Sync app redeploys your storefront site whenever you push or merge chan
-    1. Unzip the file and drag the contents into your folder. This will convert the Word and Excel files to Google Docs and Sheets.
-    </Callouts>
-
-    <Aside type="note" title="Ensure Google Drive Uploads Convert Properly">
-      To ensure Word and Excel files are automatically converted to Google Docs and Sheets, **always upload them inside a folder** rather than as individual files. The Google Drive setting for automatic conversion applies to folder uploads, but not to direct file uploads.
-    </Aside>
-
-  </TabItem>
-
-  <TabItem label="SharePoint">
+<Aside type="note" title="Demo Content">
+If you do not want demo content, you can create a file in https://da.live/#/{org}/{site}/ replacing `{org}` and `{site}` with your own. This will "reserve" the org and site for you.
+</Aside>
 
-    <Callouts>
-      1. Download the boilerplate's [starter-content-commerce.zip](https://github.com/hlxsites/aem-boilerplate-commerce/releases/download/starter-content/starter-content-commerce.zip) file.
-      1. Unzip the file and drag the contents into your SharePoint folder.
-    </Callouts>
+<Callouts>
+1. Open the [demo content clone tool](https://da.live/app/hlxsites/aem-boilerplate-commerce/tools/site-creator/site-creator).
+1. Copy your GitHub repository url to the input field and click **Create site**.
+1. Follow the prompts, and by the end the content will have been copied to your content folder.
+1. You may have to manually preview or publish the content, depending on whether you followed the prerequisites.
+</Callouts>
 
-  </TabItem>
-</Tabs>
+<Diagram caption="Initialize your storefront content">
+  ![Initialize your storefront content](images/initialize-content.webp)
+</Diagram>
 
 </Task>
 
 <Task>
 
-### Connect repo to folder content
-
-Now we need to link your GitHub storefront repo (the code) to your storefront folder (the content). Linking your code to your content enables Edge Delivery to coordinate code changes in the repo with content changes in your Google Drive or SharePoint folder. After step 1 below, all steps are the same for Google Drive and SharePoint.
+### Link repo to commerce data
 
-<Diagram caption="Replace the default boilerplate's folder URL with your content folder URL.">
-  ![Replace the default boilerplate's folder URL with your content folder
-  URL.](images/connect-code-to-content.webp)
-</Diagram>
-
-<Callouts>
-1. Copy the URL to your content folder:
+Linking your storefront repo to your commerce data can be done in two ways:
 
-    - **For Google Drive**: Open your folder and copy the URL from the browser's address bar.
-    - **For SharePoint**: Open your folder and select the **Copy link** button.
+1. Use [AEM's Configuration Service](https://www.aem.live/docs/config-service-setup) and the `PublicConfig` schema from the [Site Config API docs(https://www.aem.live/docs/admin.html#tag/siteConfig).
```

Comment:
This is still confusing. You need to specify the Commerce headers, which isn't mentioned in the provided references-- the example [default-site-config.json](https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/default-site.json) in the Commerce boilerplate shows the values required.  And the instructions in the [readme](https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/README.md) provide the API requests to use the config service.  

How does a person know when/why to use the different methods?

Reply:
Oh yeah, that should have been removed. Fixing.

Reply:
Resolved in https://github.com/commerce-docs/microsite-commerce-storefront/pull/286